src/app-layer-htp-file.c

```diff
@@ -150,6 +150,107 @@ int HTPFileOpen(HtpState *s, const uint8_t *filename, uint16_t filename_len,
     SCReturnInt(retval);
 }
 
+/**
+ * Performs parsing of the content-range value
+ *
+ * @param[in] rawvalue
+ * @param[out] range
+ *
+ * @return HTP_OK on success, HTP_ERROR on failure.
+ */
+int HTPParseContentRange(bstr * rawvalue, HtpContentRange_t *range) {
```

Comment:
style: in functions, opening bracket on new line

Reply:
ok